src/main/resources/db/migration/V13__create_table_poststed.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE postinformasjon
```

Comment:
Bare min mening her, men jeg ville foretrukket at alt som er `postinformasjon` / `PostInformasjo` isteden er `poststed` / `Poststed`, med mindre det er noe annet som er relevant for denne tabellen.

Reply:
Men blir det ikke rart å ha f.eks. 
```
data class Poststed(
    val postnummer: String,
    val poststed: String
)
```
?

Reply:
Jeg synes det er vanskelig, for jeg er enig i at postinformasjon er et dårlig ord. Vred hodet ganske lenge før jeg landet på det sist jeg lagde dette 😅 Var også innom `PostnummerOgPoststed`, men det blir jo litt langt og klønete. Jeg ser at Posten kaller det `Postnummer`, kanskje det er bedre? For vi skjønner kanskje at Postnummer-objektet inneholder et postnummer og et poststed..?

Reply:
da gjør jeg det, takk for innspill! 🙌